From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sun, 23 Nov 2014 08:43:56 +0100
Subject: FTBFS armhf: fixing ARM CPU detection

https://bugzilla.mozilla.org/show_bug.cgi?id=1066853
The armv7 CPU detection didn't match non adroid based platform with
armv7 correctly.
This patch should be not needed in further releases after Thunderbird 35.
---
 .../trunk/webrtc/system_wrappers/source/cpu_features.cc     | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/mozilla/media/webrtc/trunk/webrtc/system_wrappers/source/cpu_features.cc b/mozilla/media/webrtc/trunk/webrtc/system_wrappers/source/cpu_features.cc
index b924d773..af29d08 100644
--- a/mozilla/media/webrtc/trunk/webrtc/system_wrappers/source/cpu_features.cc
+++ b/mozilla/media/webrtc/trunk/webrtc/system_wrappers/source/cpu_features.cc
@@ -66,6 +66,19 @@ static int GetCPUInfo(CPUFeature feature) {
   (void)feature;
   return 0;
 }
+
+#if !defined(WEBRTC_GONK) && !defined(ANDROID)
+#ifdef WEBRTC_ARCH_ARM_V7
+uint64_t WebRtc_GetCPUFeaturesARM(void) {
+  return kCPUFeatureARMv7
+#ifdef WEBRTC_ARCH_ARM_NEON
+         | kCPUFeatureNEON
+#endif
+         | kCPUFeatureVFPv3;
+}
+#endif // WEBRTC_ARCH_ARM_V7
+#endif // !WEBRTC_GONK && !ANDROID
+
 #endif
 
 WebRtc_CPUInfo WebRtc_GetCPUInfo = GetCPUInfo;
